src/demo/view-configs/blin.js

```diff
@@ -0,0 +1,54 @@
+
+
+const blinName = 'Blin et al., PLoS Biol 2019';
+const blinDescription = 'Mouse blastocysts imaged by confocal microscopy';
+export const blin2019 = {
+  version: '1.0.6',
+  name: blinName,
+  description: blinDescription,
+  public: true,
+  datasets: [
+    {
+      uid: 'idr0062-blin-nuclearsegmentation/6001240',
+      name: 'idr0062-blin-nuclearsegmentation/6001240',
+      files: [
+        {
+          type: 'raster',
+          fileType: 'raster.ome-zarr',
+          url: 'https://minio-dev.openmicroscopy.org/idr/v0.3/idr0062-blin-nuclearsegmentation/6001240.zarr',
```

Comment:
May be worth making an issue for loading segmentation masks as well.  I think there is a mask in this dataset